lib/rucio/db/sqla/migrate_repo/script.py.mako

```diff
@@ -13,10 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-''' ${message} '''
+"""${message}."""
```

Comment:
Unless I am mistaken, it should be fine to remove the surrounding whitespace, see [here](https://github.com/sqlalchemy/alembic/blob/8d13dc92ddf27b95403cd73d690bc8cb94d0904b/alembic/script/base.py#L863). But maybe forcing the period outside of `message` is too much.

Reply:
Good point. I think it might be worth removing the period and adding a `# noqa: D400, D415` comment.

For reference on D400 and D415:
- https://docs.astral.sh/ruff/rules/missing-trailing-period/
- https://docs.astral.sh/ruff/rules/missing-terminal-punctuation/

These rules are pretty similar, and there's been discussion to integrate both of them in a single rule here: https://github.com/astral-sh/ruff/issues/8469, but for now we'd probably have to ignore both of them